internal/server/handlers.go

```diff
@@ -121,10 +121,19 @@ func (h *MetricHandler) GetMetric(w http.ResponseWriter, r *http.Request) {
 func (h *MetricHandler) UpdateMetricsWJSON(w http.ResponseWriter, r *http.Request) {
 	b, err := io.ReadAll(r.Body)
 	if err != nil {
-		http.Error(w, "error to read request body: "+err.Error(), http.StatusInternalServerError)
+		code := http.StatusInternalServerError
```

Comment:
По-моему, можно так:

```suggestion
		const code = http.StatusInternalServerError
```